onnxruntime/test/python/quantization/op_test_utils.py

```diff
@@ -68,6 +68,21 @@ def check_op_type_count(testcase, model_path, **kwargs):
         )
 
 
+def check_conv_per_channel_counts(testcase, model_path, node_name: str = "conv_node_quant"):
```

Comment:
check_conv_per_channel_counts is specific to Conv per-channel test. please move it to test_qdq.py